CONTRIBUTING.md

```diff
@@ -73,4 +73,61 @@ guidelines we follow:
   contribution. Finally, no branch merge commits should get into the
   *master* history.
 
+When developing Fencer, it is important to keep in mind that linting
+is performed automatically during continuous integration, i.e., before
+a pull request is approved for merging. This means that Haskell code
+will be checked against a number of stylistic or programmatic errors
+and if any of them match, the pull request will not get an
+approval. To save you from waiting on a continuous integration job to
+give a linting verdict, you can quickly lint locally before
+Git-pushing to a remote repository. When in a `nix-shell`, run:
+
+```bash
+hlint --git
+```
+
+If there are any issues with the code that match a curated list of
+linting patterns, the `hlint` tool will report them and exit with a
+non-zero status. To make sure you do not forget to lint before
+Git-pushing, you can set up a pre-push Git hook:
+
+1. Copy a script `scripts/pre-push` to the `.git/hooks`
```

Comment:
You can do `chmod +x` directly on `scripts/pre-push` and nobody would have to do this step again.

And I would suggest replacing the first step with a copy-paste-able invocation:

```
cp scripts/pre-push .git/hooks/
```

Reply:
Or it might be even better to ask contributors to create a symlink and not a copy – assuming a symlink works.

Reply:
I thought about this and did it the way I did, but I can easily change it to what you're suggesting (and I will). The `scripts/pre-push` script is an optional component of the repository, hence I intentionally made it non-executable in the directory, with which I tried to convey the idea that there's probably a reason for it. In case someone chooses to use it, they'd copy it to `.git/hooks` and make it executable.
Anyhow, I'll adopt your proposal since this is such a less important thing that needs no further discussion and any way would be good.